app/scripts/modules/kubernetes/src/v2/pipelines/stages/patchManifest/patchManifestConfig.html

```diff
@@ -0,0 +1,50 @@
+<div ng-if="ctrl.state.loaded" class="clearfix" class="clearfix">
+  <div class="alert alert-warning">
+    <strong>Warning!</strong> This stage is under active development and is subject to change.
+  </div>
+
+  <div class="container-fluid form-horizontal">
+    <h4>Resource to Patch</h4>
+    <div class="horizontal-rule"></div>
+    <kubernetes-manifest-selector selector="ctrl.$scope.stage"></kubernetes-manifest-selector>
+
+    <h4>Patch Content</h4>
+    <ng-form name="kubernetesManifestSource">
+      <stage-config-field label="Manifest Source" help-key="kubernetes.manifest.source">
+        <label class="sm-label-right">
+          <input type="radio" ng-model="ctrl.$scope.stage.source" value="{{ctrl.textSource}}">
+          {{ ctrl.textSource | robotToHuman }}
+        </label><br/>
+        <label class="sm-label-right">
+          <input type="radio" ng-model="ctrl.$scope.stage.source" value="{{ctrl.artifactSource}}">
+          {{ ctrl.artifactSource | robotToHuman }}
+        </label>
+      </stage-config-field>
+    </ng-form>
+    <kubernetes-manifest-entry ng-if="ctrl.$scope.stage.source === ctrl.textSource"
```

Comment:
By default, the "scope" of this template is `$scope` - that is, any properties of the object `$scope` can be referenced on this template without qualification. This could be `stage.source === ctrl.textSource`. The most angular-idiomatic thing would be to set `this.stage = $scope.stage` in your controller - then any time you want to reference `stage` in your controller, you'd just use `ctrl.stage`. Either way is fine, but don't do `ctrl.$scope.myProperty`.

Reply:
Setting `stage.source === ctrl.textSource` did not work for some reason. Will look into this.